README.md

```diff
@@ -939,6 +939,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Wufoo](https://www.wufoo.com/) - Quick forms to use on websites. Free plan has a limit of 100 submissions each month.
   * [formpost.app](https://formpost.app) - Free, unlimited Form to Email service. Setup custom redirect, auto response, webhooks etc for free.
   * [Formester.com](https://formester.com) - Share and embed unique-looking forms on your website—no limits on the number of forms created or features restricted by the plan. Get up to 100 submissions every month for free.
+  * [SimplePDF.eu](https://simplepdf.eu/embed) - Embed a PDF editor on your website and turn any PDF into a fillable form. Free plan allows unlimited PDFs with 3 submissions per PDF.
```

Comment:
Does it mean any pdf on your site can only be edited twice? As in this free service works only twice then you have to make a new document?

Reply:
The PDFs can be edited and filled as many times as possible (and downloaded locally). Viewing the resulting submissions by the owner of the website (who embedded the PDF editor) is limited to 3 per document.

You can see an example in action [here](https://schrutefarms.bendersej.repl.co)

Happy to rephrase any part of the explanation!

Reply:
That’s ok - well maybe clarify on the website but this is fine for here